### PR TITLE
[Synfig Studio] The canvas properties dialog now displays values correctly after being closed with the window close button.

### PR DESCRIPTION
**Pull request by rChen10**
Refresh was being called by CanvasProperties directly rather than being connected to signal_show() as demonstrated in CanvasOptions.

This is my first pull request, so any feedback is appreciated.

### CODE REVIEW

File: synfig-studio/src/gui/dialogs/canvasproperties.cpp [hunk 1]
```diff
@@ -128,7 +128,8 @@ CanvasProperties::CanvasProperties(Gtk::Window& parent,etl::handle<synfigapp::Ca
 	ok_button->signal_clicked().connect(sigc::mem_fun(*this, &studio::CanvasProperties::on_ok_pressed));
 
 	get_vbox()->show_all();
-	refresh();
+	//refresh();
```
Comment (Keyikedalube) [inline synfig-studio/src/gui/dialogs/canvasproperties.cpp:131]: You can remove that commented code.

Comment (rChen10) [reply synfig-studio/src/gui/dialogs/canvasproperties.cpp]: Alright! Thanks for the advice. I was a bit hesitant to delete code, but I suppose that is what git is for. I've updated the PR.

**Review by Keyikedalube — APPROVED** [review]

File: synfig-studio/src/gui/dialogs/canvasproperties.cpp (hunk 1 above)
Comment (Keyikedalube) [reply synfig-studio/src/gui/dialogs/canvasproperties.cpp]: You can resolve this review using the Resolve button :)

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**